### PR TITLE
Feat/db signatures

### PR DESCRIPTION
**Pull request by mjraybk07**
Adds route controllers model and schema for adding a signature to a single volunteer's single certification

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
 node_modules
 .DS_Store
 .env
-
-
-neo4j-community-3.5.8
\ No newline at end of file
+package-lock.json
```
Comment (codydaig) [inline .gitignore]: This should NOT be in the .gitignore file.

File: server/models/signature.model.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Signatures.create = ({personId, certificationId, signaturePersonId}) => {
+  return Promise.all([
+    neode.first('Person', 'id', personId),
+    neode.first('Certification', 'id', certificationId)
+  ])
+  .then(([person, certification]) => {
+    return person.relateTo(certification, 'has_certification_signature', {
+      signature_person_id: signaturePersonId,
+      signature_date: (new Date()).getTime()
```
Comment (codydaig) [inline server/models/signature.model.js]: I thought we were going to make this so the frontend could define the date signed. If none specified, default to the above.

File: server/schema/signature.schema.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const uuidv4 = require('uuid/v4');
+
+const schema = {
```
Comment (codydaig) [inline server/schema/signature.schema.js]: Is the signature it's own schema? I thought it was just a relationship.

**Comment by mjraybk07** [comment]
This PR has been replaced by PR #19.

**Pull request closed without merging**